examples/cards/cards.zig

```diff
@@ -0,0 +1,102 @@
+const std = @import("std");
+const zerl = @import("zerl");
+
+const Suit = enum {
+    clubs,
+    diamonds,
+    hearts,
+    spades,
+};
+
+const Value = enum {
+    @"2",
+    @"3",
+    @"4",
+    @"5",
+    @"6",
+    @"7",
+    @"8",
+    @"9",
+    @"10",
+    jack,
+    queen,
+    king,
+    ace,
+};
+
+const Card_Tag = enum { card };
+const Card = struct {
+    Card_Tag,
+    Value,
+    Suit,
+};
+
+const Message = union(enum) {
+    hello: *const zerl.ei.erlang_pid,
+    say: []const u8,
+    integer: i32,
+    double: f64,
+    shuffle: []const Card,
+    bye: void,
+};
+
+const Message_Tag = @typeInfo(Message).Union.tag_type.?;
+
+const Dealer_Error = enum {
+    unknown_message,
+    invalid_message,
+    some_other_error,
+};
+
+const Reply = union(enum) {
+    say: [:0]const u8,
+    halved: i31,
+    shuffled: []Card,
+    ok: void,
+    @"error": Dealer_Error,
+};
+
+const deck: [52]Card = blk: {
```

Comment:
Add a comment describing what this magic trick does.

Reply:
I object to calling it magic, but I will add a comment.